core/src/apps/benfen/sign_tx.py

```
@@ -0,0 +1,235 @@
+import binascii
+from typing import Tuple
```

Ruff (0.8.2)

`typing.Tuple` is deprecated, use `tuple` instead

(UP035)